src/entities/profile/profile.entity.ts

```diff
@@ -39,4 +50,13 @@ export class ProfileEntity {
 
   @OneToMany(() => ExperienceEntity, (experience) => experience.profile, { cascade: true })
   experience: ExperienceEntity[];
+
+  @OneToMany(() => User, (user) => user.userId)
+  @JoinColumn()
+  user: User[];
+
+  @Column({ type: 'integer' })
+  @JoinColumn()
+  @ManyToOne(() => User, (user) => user.id, { cascade: true })
+  userId: number;
```

Comment:
why we need both user and userId, according to nest documentation userId is not needed. please be more attentive to details

Reply:
I need user Id to check equality between profile table and users